zbx_redis_stats.py

```diff
@@ -10,9 +10,9 @@
 parser.add_argument('-a','--auth',dest='redis_pass',action='store',help='Redis server pass',default=None)
 args = parser.parse_args()
 
-zabbix_host = '127.0.0.1'	# Zabbix Server IP
-zabbix_port = 10051			# Zabbix Server Port
-hostname = 'redis.srv.name'	# Name of monitored server like it shows in zabbix web ui display
+zabbix_host = '127.0.0.1'       # Zabbix Server IP
+zabbix_port = 10051             # Zabbix Server Port
+hostname = 'redis.srv.name'     # Name of monitored server like it shows in zabbix web ui display
```

Comment:
Actually, it should be same name, probably it will be better to take it from command like argument, and if it's not specified, then use current node name. Something like that:

```python
import socket

hostname = arg.hostname if arg.hostname else socket.getHostName()
```